### PR TITLE
[MOB-7748] iOS constraints fix for banner img view

### PR DESCRIPTION
**Pull request by Ayyanchira**
## 🔹 Jira Ticket(s)

* [MOB-7748](https://iterable.atlassian.net/browse/MOB-7748)

## ✏️ Description

> Please provide a brief description of what this pull request does.

[MOB-7748]: https://iterable.atlassian.net/browse/MOB-7748?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Ayyanchira** [comment]
image

image

image

Current implementation

File: swift-sdk/Resources/IterableEmbeddedView.xib [hunk 1]
```diff
@@ -66,7 +66,7 @@ Description text</string>
                                     <rect key="frame" x="268" y="0.0" width="100" height="100"/>
                                     <constraints>
                                         <constraint firstAttribute="height" constant="100" id="8cm-im-EKi"/>
-                                        <constraint firstAttribute="width" constant="100" id="bVg-Ev-iLm"/>
+                                        <constraint firstAttribute="width" relation="lessThanOrEqual" constant="100" id="bVg-Ev-iLm"/>
```
Comment (Ayyanchira) [inline swift-sdk/Resources/IterableEmbeddedView.xib:69]: This is now the default behavior allowing Label and text to have more horizontal space when image is not available.

File: swift-sdk/uicomponents/IterableEmbeddedView.swift [hunk 2]
```diff
@@ -299,6 +299,9 @@ public class IterableEmbeddedView:UIView {
                 imgView.isHidden = EMimage == nil
                 imgView.isHidden = self.EMimage == nil
                 imgView.image = EMimage
+                if !imgView.isHidden {
+                    imgView.widthAnchor.constraint(equalToConstant: 100).isActive = true
```
Comment (Ayyanchira) [inline swift-sdk/uicomponents/IterableEmbeddedView.swift:303]: This overrides the behavior by placing a constant 100 width for banner images if they are available

**Review by davidtruong — APPROVED** [review]

**Review by evantk91 — APPROVED** [review]
LGTM

**Pull request merged**